### PR TITLE
Adding a git pull secret for dags in Airflow

### PR DESCRIPTION
**Pull request by wirehead**
#### What this PR does / why we need it:

Pulling Airflow DAGs via an init container is awesome, but I've got some private repos to deal with.  This PR allows users to create a secret containing a deploy key for their DAG repository.

#### Which issue this PR fixes

I didn't bother creating an issue

#### Special notes for your reviewer:

@gsemet Thanks for already having gone through 99.95% of the effort required and stepping up as a maintainer. :)

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

File: stable/airflow/Chart.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 description: Airflow is a platform to programmatically author, schedule and monitor workflows
 name: airflow
-version: 0.14.0
```
Comment (gsemet) [inline stable/airflow/Chart.yaml:3]: That’s a new feature, actually. You can bump to 0.15.0

**Review by gsemet — CHANGES_REQUESTED** [review]
Looks amazing :) I actually need it now :) just set to 0.15 and we can merge it ! Thanks !!

File: stable/airflow/Chart.yaml (hunk 1 above)
Comment (wirehead) [reply stable/airflow/Chart.yaml:3]: Fixed.

Comment (gsemet) [reply stable/airflow/Chart.yaml:3]: thanks!

**Pull request merged**